### PR TITLE
Bump serving components to 1.9

### PR DESCRIPTION
**Pull request by ReToCode**
As per title.

### CODE REVIEW

**Comment by ReToCode** [comment]
Relies on https://github.com/openshift-knative/net-istio/pull/42 and https://github.com/openshift-knative/net-kourier/pull/67

**Comment by ReToCode** [comment]
Rebased. Tests will still fail because of https://github.com/openshift-knative/serverless-operator/pull/2087#issuecomment-1551392780.

**Comment by skonto** [comment]
@ReToCode I approved the net-* PRs, let's see.

**Comment by ReToCode** [comment]
👍  Working on the PR for openshift/release to get some CI jobs for the net-* repos. Then will re-trigger here.

File: olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -832,31 +832,31 @@ spec:
...
                         value: "quay.io/maistra-dev/proxyv2-ubi8:2.3-latest"
                       - name: "IMAGE_net-kourier-controller__controller"
-                        value: "registry.ci.openshift.org/openshift/knative-v1.8.0:kourier"
+                        value: "registry.ci.openshift.org/openshift/knative-v1.9.0:kourier"
                       - name: "IMAGE_net-istio-controller__controller"
-                        value: "registry.ci.openshift.org/openshift/knative-v1.8.0:net-istio-controller"
+                        value: "registry.ci.openshift.org/openshift/knative-v1.9.0:net-istio-controller"
                       - name: "IMAGE_net-istio-webhook__webhook"
-                        value: "registry.ci.openshift.org/openshift/knative-v1.8.0:net-istio-webhook"
+                        value: "registry.ci.openshift.org/openshift/knative-v1.9.0:net-istio-webhook"
```
Comment (nak3) [inline olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:859]: It seems https://github.com/openshift/release/pull/39775 changed the image to `registry.ci.openshift.org/openshift/net-istio-controller:knative-1.9`

Comment (ReToCode) [reply olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml]: Yes, openshift-knative/hack introduces the new tagging schema. I updated the relevant parts. Lets see if tests pass now.

File: olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -832,31 +832,31 @@ spec:
...
+                      - name: "IMAGE_storage-version-migration-serving-serving-1.9__migrate"
+                        value: "registry.ci.openshift.org/openshift/knative-serving-storage-version-migration:knative-v1.9"
                       - name: "IMAGE_kourier-gateway"
                         value: "quay.io/maistra-dev/proxyv2-ubi8:2.3-latest"
                       - name: "IMAGE_net-kourier-controller__controller"
-                        value: "registry.ci.openshift.org/openshift/knative-v1.8.0:kourier"
+                        value: "registry.ci.openshift.org/openshift/net-kourier-kourier:knative-v1.9"
                       - name: "IMAGE_net-istio-controller__controller"
-                        value: "registry.ci.openshift.org/openshift/knative-v1.8.0:net-istio-controller"
+                        value: "registry.ci.openshift.org/openshift/net-istio-controller:knative-v1.9"
```
Comment (nak3) [inline olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:857]:
```suggestion
                        value: "registry.ci.openshift.org/openshift/net-istio-controller:knative-1.9"
```
I think no `v` for this image.

**Comment by nak3** [comment]
/lgtm

:tada:

**Pull request merged**